scripts/log-capture

```diff
@@ -29,20 +29,20 @@ _to_log dmesg
 _to_log dmidecode
 _to_log lspci -vvnn
 
-for DEV_NAME in $(list-harddrives | cut -d " " -f 1)
+for DEV_NAME in $(cd /sys/block ; ls);
 do
     _to_log blkid --probe /dev/${DEV_NAME}
 done
-for PART in $(blkid -d':' -f1)
+for PART in $(blkid | cut -d':' -f1)
 do
     _to_log blkid --probe ${PART}
 done
 
 _to_log ls -lR /dev
 _to_log parted -l
 _to_log lsblk --all --topology
-_to_log blkid --output-all
-_to_log ls -l /sys/firmware/efi/efivars
+_to_log lsblk --output-all
```

Comment:
This results in so many columns, mainly unreadable on even a desktop monitor. Shall we print this in json format?

Reply:
Fixed